src/eos-crash-metrics.c

```diff
@@ -179,6 +306,12 @@ main (int argc, char **argv)
       return EXIT_FAILURE;
     }
 
+  if (g_str_has_prefix (path, "/app/bin"))
+    {
+      g_message ("%s is likely a Flatpak, get information", path);
+      flatpak_info = get_flatpak_info (path, &error);
```

Comment:
There should be some error checking here; if `get_flatpak_info()` fails, I think we should print the error message in a warning, and `return EXIT_FAILURE;`

Reply:
Yes done.